clash/the_hunt.tex

```diff
@@ -151,16 +151,20 @@ \subsection*{\MakeUppercase{Additional Rules}}
   \centering
   \medskip
 
-  \newcommand{\bronze}[0]{\svgunit[12]{bronze}}
-  \newcommand{\silver}[0]{\svgunit[12]{silver}}
-  \newcommand{\golden}[0]{\svgunit[12]{golden}}
-  \newcommand{\azure}[0]{\svgunit[12]{azure}}
+  \let\origbronze\bronze
```

Comment:
please add a comment

Reply:
What kind of comment? That's just redefining the commands with a non-default arg value.

Reply:
I see. While I was typing this, I didn't know yet that its scope is limited, and didn't understand why you need to repeat this thing/redefine them every time